<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/Frame 62.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>428</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 150px;
height: 40px;
left: 180px;
top: 428px;

border: 2px solid #4557A3;
box-sizing: border-box;
border-radius: 20px;

font-family: Open Sans;
font-style: normal;
font-weight: regular;
font-size: 12px;
line-height: 20px;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="User">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>369</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 369px;

background: rgba(0, 0, 0, 0.02);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>452</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 369px;

background: rgba(0, 0, 0, 0.02);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sign">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>550</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 550px;

background: #526BD6;
border-radius: 10px;

font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 18px;
line-height: 25px;
/* identical to box height */


color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Number">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>499</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 369px;

background: rgba(0, 0, 0, 0.02);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass_2">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>416</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 369px;

background: rgba(0, 0, 0, 0.02);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="User_2">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>333</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 369px;

background: rgba(0, 0, 0, 0.02);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>327</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>repear.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="car">
   <property name="geometry">
    <rect>
     <x>1122</x>
     <y>41</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Group 85.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="delivery">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>41</y>
     <width>50</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Delivery.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="shop">
   <property name="geometry">
    <rect>
     <x>1275</x>
     <y>41</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/ant-design_shopping-outlined.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="character">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>138</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Characteristics.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Appearance">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>141</y>
     <width>60</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Appearance.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="back">
   <property name="geometry">
    <rect>
     <x>877</x>
     <y>141</y>
     <width>111</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/butt.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="x5">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>246</y>
     <width>388</width>
     <height>522</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/cat/bmw.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="carEdit">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>230</y>
     <width>711</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/stock.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stock">
   <property name="geometry">
    <rect>
     <x>877</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/stockYES.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="rims2">
   <property name="geometry">
    <rect>
     <x>973</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/2NO.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="rims3">
   <property name="geometry">
    <rect>
     <x>1068</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/3NO.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="exit">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>44</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/delivery/exit.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stockCh">
   <property name="geometry">
    <rect>
     <x>877</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/stockCharYES.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stage1">
   <property name="geometry">
    <rect>
     <x>973</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Stage1NO.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stage2">
   <property name="geometry">
    <rect>
     <x>1069</x>
     <y>449</y>
     <width>84</width>
     <height>129</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/Stage2NO.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="buy">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>645</y>
     <width>175</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/!_DOCUMENTS_!/Desktop/Курсач/car/add.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="accept">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>235</y>
     <width>403</width>
     <height>544</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>31</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 330px;
height: 50px;
left: 773px;
top: 550px;

background: #3C4A88;
border-radius: 10px;

font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 9px;
line-height: 12px;
/* identical to box height */


color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>709</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 106px;
height: 33px;

font-family: Open Sans;
font-style: normal;
font-weight: 600;
font-size: 24px;
line-height: 33px;
/* identical to box height */


color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
